source/_integrations/immich.markdown

```diff
@@ -28,7 +28,7 @@ You need to [obtain the API key](https://immich.app/docs/features/command-line-i
 
 ### API key permissions
 
-For full functionality, enable the `album.read` permission when creating your API key. Without this permission, the media source integration will not work, but all monitoring sensors will continue to function normally.
+For full functionality, enable the `album.read` and the `asset.upload` permission when creating your API key. Without this permission, the media source integration will not work, but all monitoring sensors will continue to function normally.
```

Comment:
with immich [1.136.0](https://github.com/immich-app/immich/releases/tag/v1.136.0) the [api key permission system](https://github.com/immich-app/immich/pull/20113) has been overhauled,  so this section needs a bit more love in a coming up PR